app/gui/view/component-browser/component-list-panel/grid/src/layout.rs

```diff
@@ -132,12 +132,7 @@ impl Layout {
     ) -> Self {
         let local_scope_rows = local_scope_entry_count.div_ceil(COLUMN_COUNT);
         let filtered_groups = groups.map(|mut g| {
-            g.drain_filter(|g| {
```

Comment:
Do we know why the code had a special case for `original_height == 2`? I'm just wondering if this change will result in some subtle breakage.

Reply:
It was only for a warning. I've debugged missing Component groups and this log sneaked review then.